lib/kms.js

```diff
@@ -6,6 +6,28 @@ import * as brHttpsAgent from 'bedrock-https-agent';
 import {getAppIdentity} from 'bedrock-app-identity';
 import {KeystoreAgent, KmsClient} from '@digitalbazaar/webkms-client';
 
+bedrock.events.on('bedrock.init', () => {
+  if(!(bedrock.config && bedrock.config.tokenizer &&
+    bedrock.config.tokenizer.kms && bedrock.config.tokenizer.kms.baseUrl)) {
```

Comment:
If you wanted to add a node 14+ restriction to package.json and test actions, you could use optional chaining to make this pattern nicer. (I think bedrock.config always exists?)
```suggestion
  if(!bedrock.config.tokenizer?.kms?.baseUrl) {
```